### ISSUE TITLE
Disable repeated video data delete confirmation dialogue in Desktop

### ISSUE THREAD

**Issue by mattdawkins** [Kitware/dive#994]
When dealing with many sequences, deleting data is a pain with confirmation dialogues on every delete. The information in the current delete dialogue should maybe only appear once, on the first user delete, or would be better suited for ? tooltip next to the delete button the user could hover over?

**Comment by subdavis** [comment]
This is why we added a setting to turn of delete confirmation.  Did you use that?

**Comment by waxlamp** [comment]
@subdavis, is there anything blocking us from implementing checkbox-style mass delete on desktop?

**Comment by subdavis** [comment]
Checkbox-style mass delete would not resolve this issue.  

The ask is to also delete media from outside the DIVE project directory, which I'm very wary of doing.

**Comment by waxlamp** [comment]
> Checkbox-style mass delete would not resolve this issue.
> 
> The ask is to also delete media from outside the DIVE project directory, which I'm very wary of doing.

I would agree with your wariness. Something halfway out-of-band might be good: remove the "deleted" items from DIVE's awareness, and generate a report of all filenames and give to the user so they can do an `xargs | rm` or equivalent by their own hand.
